ui_testing/pages/order_page.py

```diff
@@ -308,6 +315,39 @@ def get_suborder_data(self, sub_order_index=0):
         order_data['suborders'] = suborders_data
         return order_data
 
+    def construct_main_order_from_table_view(self, order_row=None):
```

Comment:
this function needs to be moved to orders_page, since it is related to orders table page not the form page